src/modules/Cron/html_admin/mod_cron_settings.html.twig

```diff
@@ -63,6 +62,26 @@
                 </tbody>
             </table>
         </div>
+        <div class="card-body container">
+            {% set params = admin.extension_config_get({ 'ext': 'mod_cron' }) %}
+            <form method="post" action="{{ 'api/admin/extension/config_save'|link }}" class="api-form" data-api-msg="{{ 'Configuration updated'|trans }}">
+                <input type="hidden" name="CSRFToken" value="{{ CSRFToken }}">
+                <input type="hidden" name="ext" value="mod_cron">
+                <div class="row">
+                    <div class="col-3">
+                        <label>{{ 'Enable the cron API endpoint for guests'|trans }}</label>
+                    </div>
+                    <div class="col-1">
+                        <label class="form-check form-switch">
+                            <input class="form-check-input" name="guest_cron" type="checkbox" {% if params.guest_cron %} checked="checked"{% endif %}>
+                        </label>
+                    </div>
+
+                    <p class="text-muted">{{ 'If enabled, this URL can be used to execute the cronjob:'|trans }} {{ 'api/guest/cron/run'|link }}</p>
```

Comment:
We should at least consider some basic / minimal support for at least a  random string and don't allow unauthenticaetd requests 

/api/quest/cron/run/?key=random_string_that_is_xx_chars_long is a beter protection then nothing ...

Reply:
I think that's pretty overkill for a few reasons.

1. This is completely disabled by default and must be explicitly enabled. It is only here for the small handful of people who may for whatever reason may not be able to rely on just CLI-based cron.
2. If someone genuinely wanted an implementation that's authenticated, it already exists. There's an admin API endpoint to run cron and they could make a dummy account with as little permissions as possible (not that I would suggest doing so, as I believe an extra account hooked up to an external service poses a greater risk than calling cron without authentication).
3. The only real "threat" this could pose is if someone hammered the endpoint to make cron run a bunch to slow down the server. But that shouldn't be any more effective than doing any other bulk request. Even then, I added a basic check to prevent it from being run more than 1 time every minute.